android-components/components/feature/autofill/src/main/java/mozilla/components/feature/autofill/response/fill/AuthFillResponseBuilder.kt

```diff
@@ -101,9 +102,20 @@ internal fun FillResponse.Builder.setAuthentication(
     inlinePresentation: InlinePresentation? = null,
     presentation: RemoteViews,
 ): FillResponse.Builder {
-    return if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R && inlinePresentation != null) {
+    return if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
+        val presentations: Presentations.Builder = Presentations.Builder()
+        presentations.apply {
+            inlinePresentation?.let {
+                setInlinePresentation(it)
+            }
+            setMenuPresentation(presentation)
+        }.build()
+        this.setAuthentication(ids, authentication, presentations.build())
```

Comment:
Actually, it is the other way around, because `presentations` is a Builder, and it could not hold a `Presentation`.
The first build does nothing, I will remove it.